src/zenpos.jl

```diff
@@ -0,0 +1,78 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _zenpos{T<:AbstractFloat}(jd::T, latitude::T, longitude::T)
+
+    lst = ct2lst(longitude, jd)
+    ra = lst*pi/12
+    dec = ra*0.0 + deg2rad(latitude)
```

Comment:
They are no more there, thanks for spotting them! I was too young, in part of old AstroLib code there are really bad examples, e.g., use of `1.0` in place of `1` ;-)